backstop/scenarios/user-icon.js

```diff
@@ -21,5 +24,6 @@ module.exports = [
   scenario('State hover animated', {
     actions: [hover('#test-animated-1')],
     selectors: ['#demo-animated-1'],
+    postInteractionWait: 1000,
```

Comment:
I spent a while playing around with this and still can't get it to pass. I don't understand anymore.

Reply:
Yeah..that got weird. Lets leave it out for this PR not to hang and maybe @bentleyvk could have a try when he is back..